_data/toc.yml

```diff
@@ -60,6 +60,8 @@
           url: "geyser/extensions/#creating-commands-with-geyser-extensions"
     - title: Global API
       url: "geyser/global-api/"
+    - title: Geyser and Secure ChatGeyser and Secure Chat
```

Comment:
This page wasn't added to the TOC on purpose, instead, it's linked to in e.g. common issues